avocado/utils/configure_network.py

```diff
@@ -92,3 +96,84 @@ def unset_ip(interface):
         return True
     except Exception as ex:
         raise NWException("ifdown fails: %s" % ex)
+
+
+class HostInfo:
+    """
+    class for host function
+    """
+
+    def ping_check(self, interface, peer_ip, count, option=None, flood=False):
+        """
+        Checks if the ping to peer works.
+        """
+        cmd = "ping -I %s %s -c %s" % (interface, peer_ip, count)
+        if flood is True:
+            cmd = "%s -f" % cmd
+        elif option is not None:
+            cmd = "%s %s" % (cmd, option)
+        if process.system(cmd, shell=True, verbose=True,
+                          ignore_status=True) != 0:
+            return False
+        return True
+
+    def set_mtu_host(self, interface, mtu):
+        """
+        set mtu size in host interface
+        """
+        cmd = "ip link set %s mtu %s" % (interface, mtu)
+        try:
+            process.system(cmd, shell=True)
```

Comment:
what about if the command error out, it needs to handle and raise Exception as mtu size can not be set